community/coreutils/build

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+export CC="gcc -static"
+
+export FORCE_UNSAFE_CONFIGURE=1
+
+./configure \
+    --prefix=/opt/gnu/coreutils \
```

Comment:
Can we not install to `/usr/bin` now as all programs are prefixed with `g`?

Reply:
ERRG ARGH ACK Okay  got it!!!